pkg/cluster/config.go

```diff
@@ -0,0 +1,27 @@
+package cluster
+
+import (
+	anywherev1 "github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+)
+
+type Config struct {
```

Comment:
does bundle considered "external data objects" and will be only in `cluster.spec`?

Reply:
Yeah, correct